### PR TITLE
Hotfix for 0.56.2

### PR DESCRIPTION
**Pull request by sklam**
as titled

### CODE REVIEW

**Comment by sklam** [comment]
`numba_smoketest_cpu_yaml_126`

File: CHANGE_LOG [hunk 1]
```diff
@@ -19,6 +19,12 @@ Pull-Requests:
 * PR `#8411 <https://github.com/numba/numba/pull/8411>`_: update version support table for 0.56.1 (`esc <https://github.com/esc>`_)
 * PR `#8412 <https://github.com/numba/numba/pull/8412>`_: Create changelog for 0.56.1 (`Andre Masella <https://github.com/apmasell>`_)
 * PR `#8413 <https://github.com/numba/numba/pull/8413>`_: Fix Azure CI for NumPy 1.23 and use conda-forge scipy (`Siu Kwan Lam <https://github.com/sklam>`_)
+* PR `#8414 <https://github.com/numba/numba/pull/8413>`_: Hotfix for 0.56.2 (`Siu Kwan Lam <https://github.com/sklam>`_)
+*
```
Comment (sklam) [inline CHANGE_LOG:23]:
```suggestion

```

File: CHANGE_LOG [hunk 2]
```diff
@@ -19,6 +19,11 @@ Pull-Requests:
 * PR `#8411 <https://github.com/numba/numba/pull/8411>`_: update version support table for 0.56.1 (`esc <https://github.com/esc>`_)
 * PR `#8412 <https://github.com/numba/numba/pull/8412>`_: Create changelog for 0.56.1 (`Andre Masella <https://github.com/apmasell>`_)
 * PR `#8413 <https://github.com/numba/numba/pull/8413>`_: Fix Azure CI for NumPy 1.23 and use conda-forge scipy (`Siu Kwan Lam <https://github.com/sklam>`_)
+* PR `#8414 <https://github.com/numba/numba/pull/8413>`_: Hotfix for 0.56.2 (`Siu Kwan Lam <https://github.com/sklam>`_)
+Version 0.56.1 (1 September, 2022)
```
Comment (sklam) [inline CHANGE_LOG:23]:
```suggestion
* PR `#8414 <https://github.com/numba/numba/pull/8413>`_: Hotfix for 0.56.2 (`Siu Kwan Lam <https://github.com/sklam>`_)

Version 0.56.1 (1 September, 2022)
```

File: CHANGE_LOG [hunk 3]
```diff
@@ -19,6 +19,12 @@ Pull-Requests:
 * PR `#8411 <https://github.com/numba/numba/pull/8411>`_: update version support table for 0.56.1 (`esc <https://github.com/esc>`_)
 * PR `#8412 <https://github.com/numba/numba/pull/8412>`_: Create changelog for 0.56.1 (`Andre Masella <https://github.com/apmasell>`_)
 * PR `#8413 <https://github.com/numba/numba/pull/8413>`_: Fix Azure CI for NumPy 1.23 and use conda-forge scipy (`Siu Kwan Lam <https://github.com/sklam>`_)
+* PR `#8414 <https://github.com//pull/8413>`_: Hotfix for 0.56.2 (`Siu Kwan Lam <https://github.com/sklam>`_)
```
Comment (esc) [inline CHANGE_LOG:22]:
```suggestion
* PR `#8414 <https://github.com/numba/numba/pull/8413>`_: Hotfix for 0.56.2 (`Siu Kwan Lam <https://github.com/sklam>`_)
```

**Review by esc — CHANGES_REQUESTED** [review]
The URL appears to have been shortened.

**Review by esc — APPROVED** [review]
Can be merged when public CI passes.

**Pull request merged**